### PR TITLE
Fix spelling and grammar in doc

### PR DESCRIPTION
**Pull request by LukaszKokot**
Just a few spelling, grammar and punctuation fixes I've found while reading the documentation.

### CODE REVIEW

File: INSTALL-DOCKER.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ You can now access these from your browser:
 - Hit `Ctrl+C` to turn off containers.
 - Type `docker-compose up` to start them up again.
 - You might want to run containers as daemon: `docker-compose up -d` and then read the logs only from "trustroots" container: `docker-compose logs trustroots`. You can use a helper script: `./scripts/docker/up.sh`
-- If you see a lot of `Run migrate DB script on update` coming from Nginx container, it means NodeJS isn't up yet or it has stopped due error. You should see "Trustroots server is up and running now." once Trustroots is really running.
+- If you see a lot of `Run migrate DB script on update` coming from Nginx container, it means NodeJS isn't up yet or it has stopped due tp errors. You should see "Trustroots server is up and running now." once Trustroots is really running.
```
Comment (simison) [inline INSTALL-DOCKER.md]: Typo "tp"

File: README.md [hunk 2]
```diff
@@ -56,8 +56,7 @@ Trustroots was built on the meanjs.org version.
 ## Further reading
 
 - [Development](https://github.com/Trustroots/trustroots/wiki/Development) wiki page.
-- [INSTALL.md](https://github.com/Trustroots/trustroots/blob/master/INSTALL.md) also contains good information about running 
-s and creating mock data. 
+- [INSTALL.md](https://github.com/Trustroots/trustroots/blob/master/INSTALL.md) also contains good information about running and creating mock data.
```
Comment (simison) [inline README.md]: This s+newline was probably accidental

File: INSTALL-DOCKER.md (hunk 1 above)
Comment (LukaszKokot) [reply INSTALL-DOCKER.md]: Man, I made a typo in my typo PR :sweat_smile: Will fix that, good catch!

**Comment by guaka** [comment]
@LukaszKokot wanna join us on our slack? do you have an email I can send an invite to?
(you can send me an email to kasper at trustroots dot org)

**Comment by LukaszKokot** [comment]
@guaka @simison I've updated the requested fix :-)

**Comment by simison** [comment]
💪 magnifique

**Pull request merged**